verilog/tools/kythe/kythe_facts_extractor.h

```diff
@@ -132,8 +133,16 @@ class KytheFactsExtractor {
   // VName.
   VName ExtractVariableReference(const IndexingFactNode& node);
 
+  // Extracts kythe facts from a function or task node and returns its VName.
+  VName ExtractFunctionOrTask(const IndexingFactNode& function_fact_node);
```

Comment:
Would it make more sense to separate these into `ExtractFunctionDefinition` and `ExtractTaskDefinition`?  Even if they are similar today?

Reply:
I don't think so because Kythe has nothing called `task` so the best thing is to insure that tasks are treated as functions because this is the best representation for it in kythe.